front/lib/swr/labs.ts

```diff
@@ -103,7 +103,7 @@ export function useUpdateTranscriptsConfiguration({
   transcriptConfigurationId: number;
 }) {
   return async (data: Partial<PatchTranscriptsConfiguration>) => {
-    const response = await fetch(
+    return fetch(
```

Comment:
If you look at swr/spaces.ts as an example we generally embed the notification system in the SWR doXXX function. Would be nice to align the patters here?